infrastructure/src/main/java/generator/Generator.java

```diff
@@ -0,0 +1,22 @@
+package generator;
+
+import com.mifmif.common.regex.Generex;
+import regex.RegexValidator;
+
+public class Generator {
+    public static String generateToken() {
+        return new Generex(RegexValidator.TOKEN_REGEX).random();
+    }
+
+    public static String generateElasticsearchIndexName() {
+        return new Generex(RegexValidator.ES_INDEX_NAME_REGEX).random();
+    }
+
+    public static String generateName() {
+        return new Generex(RegexValidator.NAME_REGEX).random();
+    }
+
+    public static String generateMessage(int messageLength) {
+        return new Generex("[A-Za-z]{"+messageLength+"}").random();
+    }
```

Comment:
`generateName`, `generateMessage` are for tests
`generateToken`, `generateElasticsearchIndexName` are for production code

I think it makes more sense to separate them. Plus, give them more specific name rather than "Generator"

Reply:
Changed.